### PR TITLE
Deploy Budget APP

### PR DESCRIPTION
**Pull request by Lichapa**
## Deploy Budget APP

In this milestone, I have done the following:

- Set up the repository and tools.
- Create your models according to the ERD diagram with validations and tests.
- Implement the authentication and authorization.
- Create the page to add a category.
- Create the home page.
- Create the page to add a transaction.
- Create the transactions page.
- Create tests with capybara.
- Deploy the project, and test for final details.
- Record a video for your project.
- Create a good README .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # Budget Rails app
 
 This project is for developing a budget app using rails
+
+- [Live Demo - Heroku](https://intense-harbor-49660.herokuapp.com/)
+- [Video Presentation](https://www.loom.com/share/6d01e431f7a0467dad0c6122f6cbd497)
```
Comment (keneogu) [inline README.md:6]: In your loom video, you didn't talk about the purpose of the project and about a piece of code you found interesting or proud of making, please redo another video and submit showing a clear and objective description about the project.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
 # Budget Rails app
 
 This project is for developing a budget app using rails
+
+- [Live Demo - Heroku](https://intense-harbor-49660.herokuapp.com/)
```
Comment (keneogu) [inline README.md:5]: I was able to access your app once and it developed a fault. please upload the recent commit or check for the error.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @Lichapa, 

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history
- Good PR description
- Your Design is looking good

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] fix your heroku live link. I logged me in once and I wasn't able to login again.
- [ ] Your password verification is not working it logs me in irrespective of what I use as password verification.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @Lichapa 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
- [ ] As the previous reviewer requested, please make sure that your live demo version is working properly, currently it's crashing while creating new categories.
- [ ] As the previous reviewer requested, please add acknowledgement to the design author in your README for keeping it professional. 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Lichapa 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done making the requested changes. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**